### DISCUSSION TITLE
熊仔開的饒舌課程沒有用??!!來這裡聽我娓娓道來 從0開始做一首歌 by lil Bo

### DISCUSSION THREAD

**Discussion by CPW0919** [1101-GS4719-C/Python-Community-GS4719-C#1294] [Ideas]
隨著大嘻哈時代2的結束，越來越多人對台灣的嘻哈文化更有興趣，也有很多人迫不及待想要自己創作一首饒舌歌曲來抒發自己的想法，其實要從0開始完成一手饒舌歌曲沒有你想像的那麼困難，我覺得熊仔的那個線上課程有點名過其實，其實大可不必去上課。
          
我從高一開始寫歌，寫到現在大二，中間雖然有停滯的時間例如學測指考，但其實創作是部會停滯的，任何時候腦中就有可能突然有一些靈感或是韻腳，也可以隨手記下。創作源於生活，走在路上 騎車出門 在家洗澡 任何地方都有可能造就你的創作靈感，只要對生活的敏感度提升，到處觀察四周的人事物，也會對你的創作能力有一定的幫助。
          
以上都在說要怎麼創作，現在再來聊聊如何做一手饒舌歌，現在饒舌的範圍越來越模糊了，不過饒舌歌曲不外乎就是兩個要素:rhyme and flow，第一個是rhyme也就是韻腳，蠻多新手都不知道要怎麼快速想韻腳，但其實也沒甚麼捷徑，只要每次看到一個詞彙時，下意識地想想看有哪些詞有相同的聲符 韻符 甚至是平仄，就可以對韻腳能力有很大的幫助，其實就是多練習啦!
         
第二個是flow，什麼是flow呢， 其實就是一段話的節奏，包括一些抑揚頓挫或是聲調轉換，然而練習flow變換的方式有很多種，其中一個就是，我現在隨便講一句例如: 今天天氣很好 大家好!，你就可以練習用不同的方式說這段話，像是把重音提前 語速加快減慢之類的，多練習後可以再練習搭配音樂，跟著音樂的重拍跟拍點一起，多練習後就可以對flow有更多的進步。
結論，我覺得多聽格再搭配youtube再多練習就可以學好怎麼寫歌了，要步要買線上課程呢 我語帶保留。
     
以下我的歌 歡迎聽聽看
[https://drive.google.com/file/d/1dWgmuBAiTEpzFDD8K5l7TFUZaRlX8y_J/view?usp=share_link](url)**

**Comment by Tzuhsuannnnn** [comment]
雖然我是熊仔粉絲看到lil Bo這樣講有點不給過，但我也從這篇文學到很多跟製作自己第一首rap 有關的知識，希望之後可以去參加lil Bo 的演唱會!

**Comment by Chiuhsianglin** [comment]
題外話我昨天在看REX跟OD的直播，REX開過隧道的時候就問說隧道漏水是正常的嗎，OD就騙他說因為隧道要榻了，然後REX就很緊張的說真假，可是前面限速ㄟ 太可愛了哈哈哈哈哈

**Comment by a3240281370** [comment]
挺一個政大黑音下一屆社長lil bo
